src/languages/es.js

```diff
@@ -820,6 +820,17 @@ export default {
     },
     emojiPicker: {
         skinTonePickerLabel: 'Elige el tono de piel por defecto',
+        headers: {
+            frequentlyUsed: 'Usado frecuentemente',
```

Comment:
Out of curiosity, how did you get these translations? Did you just find a list of categories elsewhere or use something like google translate?

Reply:
I used Google translate and also compared it with Slack. Except for Frequently used, rest all matched with Slack. 

Slack uses `Most used` Spanish translation for "Frequently used"

Reply:
I asked Ionatan to take a look since he speaks Spanish natively. As soon as you fix his comment this PR looks good to go!

Reply:
I should have tagged him earlier, I'm sorry about that! Will keep in mind next.